capstone/src/main/webapp/emailTemplates/announcement-email.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <script src="script.js"></script>
+    <style>
+      .email-content {
+        border: 2px double #000;
+        color: #000080;
+        font-size: 20px;
+        padding: 20px;
+        background: #f5fcfc;
+      }
```

Comment:
If all emails are intended to have the same style, is it possible to pull this out in a separate file? If it isn't that's fine.

Reply:
From what I read, Gmail does not make it possible to import external css files anymore, so I think inline css would be the only way :( This is one of the articles I read: https://www.litmus.com/blog/do-email-marketers-and-designers-still-need-to-inline-css/